tax_credit/tests/test_plotting_functions.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2014--, tax-credit development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+
+from unittest import TestCase, main
+import json
+from io import StringIO
+import pandas as pd
+from biom.cli.util import write_biom_table
+from biom import Table
+from os import remove
+from tax_credit.plotting_functions import (calculate_linear_regress,
+                                           per_level_kruskal_wallis,
+                                           make_distance_matrix,
+                                           per_method_distance,
+                                           within_between_category_distance,
+                                           isolate_top_params)
+
+
+class EvalFrameworkTests(TestCase):
+
+    def setUp(self):
+        self.table1 = pd.DataFrame([(n.split('\t'))
+                                    for n in table1.split('\n')],
+                                   columns=['group', 'dataset','level',
+                                            'x', 'y', 'c'],
+                                   dtype=float)
```

Comment:
Personally I would make `table1` a private variable in `EvalFrameworkTests`, rather than defining it at module level.